doc/api/http2.md

```diff
@@ -252,7 +253,7 @@ and would instead register a handler for the `'stream'` event emitted by the
 ```js
 const http2 = require('http2');
 
-// Create a plain-text HTTP/2 server
+// Create a simple HTTP/2 server
```

Comment:
I think the term `plain-text` is more specific.

Reply:
How about using the term `unencrypted` over `plain-text`?

Reply:
I'd be good with that.

Reply:
Sounds good to me 👍